parser_test.go

```diff
@@ -253,8 +254,13 @@ func TestGetAllGoFileInfo(t *testing.T) {
 	err := p.getAllGoFileInfo(searchDir)
 
 	assert.NoError(t, err)
-	assert.NotEmpty(t, p.files["testdata/pet/main.go"])
-	assert.NotEmpty(t, p.files["testdata/pet/web/handler.go"])
+	if runtime.GOOS == "windows" {
```

Comment:
Don't need to specify os for CI.

Reply:
@easonlin404 I known that, however,it is necessary for some developers preferring windows IDEs  like me, to run these tests first before comitting.  Are you sure to remove it?

Reply:
In my point, I think the better way is to use [filepath](https://golang.org/pkg/path/filepath/) to determine instead of using `runtime.GOOS`. 😄

Reply:
and thanks for your miner fix. @sdghchj